nixpkgs/nixpkgs.bzl

```diff
@@ -142,6 +145,26 @@ def nixpkgs_package(*args, **kwargs):
   else:
     _nixpkgs_package(*args, **kwargs)
 
+def _execute_or_fail(repository_ctx, arguments, failure_message = "", *args, **kwargs):
```

Comment:
Does `*args` work after `failure_message=`? I’m forgot most of the details of this pythonism.

Reply:
Yes.